python/py-xxhash/Portfile

```diff
@@ -4,26 +4,27 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-xxhash
-version             3.0.0
+version             3.4.1
 categories-append   devel
 platforms           darwin
 license             BSD
 maintainers         nomaintainer
+supported_archs     arm64 x86_64
```

Comment:
can you elaborate on why this now only supports these architectures? Previous version appear to build fine on all older macOS versions including systems that are i386.

Reply:
I added that line because a shared library is built, and I thought it was necessary. But I only have access to an `arm64` machine now.

What should the correct setting be?

Reply:
perhaps nothing so that it will be built for all platforms that MacPorts supports? Typically, "platforms" is only set specifically if it's obvious that something will only work on certain platforms.

Reply:
Okay, thank, I removed `supported_archs`.